README.md

```diff
@@ -2,8 +2,33 @@
 
 ## installation:
 
```

Comment:
maybe here a sentece explaining that you should have anaconda installed and either create an env or use an existing one?

Reply:
I added some text here. Have a look please.